explainability-service/src/main/java/org/kie/trustyai/service/data/utils/ModelMeshInferencePayloadReconciler.java

```diff
@@ -36,6 +36,14 @@ public class ModelMeshInferencePayloadReconciler extends InferencePayloadReconci
     @Inject
     Instance<DataSource> datasource;
 
+    private String standardizeModelId(String inboundModelId) {
+        if (inboundModelId != null && inboundModelId.contains("__isvc")) {
+            return inboundModelId.split("__isvc")[0];
+        } else {
+            return inboundModelId;
+        }
+    }
+
```

Comment:
@RobGeada Would it be ok to add a few simple unit tests? e.g. model name with
- no `__isvc`
- standard name with `__isvc`
- multiple `__isvc`